authentication/auth.go

```diff
@@ -36,42 +37,56 @@ func (e *KeyLoadError) Error() string {
 	return e.Op + ": " + e.Err
 }
 
-func loadEncryptionKey() (*rsa.PublicKey, *KeyLoadError) {
+func loadEncryptionKey() (*rsa.PublicKey, string, *KeyLoadError) {
 	encryptionKeyPath := settings.Get("JWT_ENCRYPTION_KEY_PATH")
 
 	keyData, err := ioutil.ReadFile(encryptionKeyPath)
 	if err != nil {
-		return nil, &KeyLoadError{Op: "read", Err: "Failed to read encryption key from file: " + encryptionKeyPath}
+		return nil, "", &KeyLoadError{Op: "read", Err: "Failed to read encryption key from file: " + encryptionKeyPath}
```

Comment:
Rather than returning three arguments, it might be neater to return a type, something like:
```
type keyResult struct {
    key rsa.PublicKey
    kid string
}
```